docs/reference/document_tree.rst

```diff
@@ -220,7 +220,7 @@ like this:
     .. code-block:: jinja
 
         {% render(controller(
-            'sonata.admin.doctrine_phpcr.tree_controller:treeAction',
+            'sonata.admin.doctrine_phpcr.tree_controller::tree',
```

Comment:
don't we still need the "Action" suffix with the double colon notation?

Reply:
according to https://symfony.com/doc/current/best_practices/controllers.html#controller-action-naming the `Action` suffix is deprecated. however, the thing here must match the method name, and you are not adjusting the method name in this PR.

changing the method name is a BC break. i suggest we make a ticket here for this, first adding the method and make the `Action` methods call those method and trigger a deprecation warning, then later remove it when we do the next major version. but its a bit out of scope of this PR, and this PR is big enough as is.

Reply:
created #537 and changed the doc back to use `Action`